src/main/java/com/ecwid/consul/v1/QueryParams.java

```diff
@@ -47,6 +47,13 @@ public QueryParams(long waitTime, long index) {
 		this.index = index;
 	}
 
+    public QueryParams(String datacenter, long waitTime, long index) {
```

Comment:
Agree, we could do the same for PutParams. Next time, when I make a change, will change it to builder pattern.